### ISSUE TITLE
stderr_read_cb writes to sys.stdout instead of sys.stderr

### ISSUE THREAD

**Issue by robvdl** [ybrs/single-beat#24]
Is this a copy paste error? Should stderr_read_cb not write to sys.stderr instead?

```python
def stdout_read_cb(self, data):
    sys.stdout.write(data.decode())

def stderr_read_cb(self, data):
    sys.stdout.write(data.decode())
```

**Comment by robvdl** [comment]
Also I am wondering if these callbacks should be flushing.

I was noticing some odd behaviour that lines produced by the celery child process would not appear until the parent process single-beat would produce a log line.

It seems by inserting flush statements in the stdoout_read_cb and stderr_read_cb methods it has fixed this.

```python
def stdout_read_cb(self, data):
    sys.stdout.write(data.decode())
    sys.stdout.flush()

def stderr_read_cb(self, data):
    sys.stderr.write(data.decode())
    sys.stderr.flush()
```

I did try setting PYTHONBUFFERED=1 in my supervisor config file, so I wasn't sure what was going on, but it seems adding flush statements to the callbacks at least fixed it for me. But that is a separate issue really.

**Comment by robvdl** [comment]
See either #27 or #28

**Comment by ybrs** [comment]
can you try it with `PYTHONUNBUFFERED=1` environment variable ? 

Indeed, it's buffered, but not sure it's a good idea to flush per each line. But also in simple cases - like log lines - you shouldn't notice it. (At least I never noticed when tailing logs)

What operating system and python version are you using  - maybe something to do with it. Also are you running it in docker/compose kind of containers ? 

Also for the first question, yes that's looks like a mistake, probably a typo, we shouldn't be redirecting stderr to stdout - but I'll double check.

**Comment by ybrs** [comment]
I merged #27, so closing this issue for now. Will release a new version soon.
